framework/src/main/java/org/checkerframework/common/aliasing/AliasingVisitor.java

```diff
@@ -271,7 +265,37 @@ private boolean canBeLeaked(Tree exp) {
         AnnotatedTypeMirror type = atypeFactory.getAnnotatedType(exp);
```

Comment:
You should also update the javadoc with the change in behavior.
Do you understand what `canBeLeaked` expresses? The documentation is rather unclear.

Reply:
I have changed the documentation for the canBeLeaked method in my latest commit